src/Api/Managers/NotificationManager.cs

```diff
@@ -1,65 +1,87 @@
-﻿using BLL.Managers;
-using BOL.Data;
+﻿using BOL.Data;
 using BOL.Models;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace BLL.Managers
 {
     public class NotificationManager : BaseManager
     {
         public NotificationManager(CmsDbContext dbContext) : base(dbContext) { }
 
-        public virtual IEnumerable<Notification> CurrentUser(User userId)
+        public virtual IEnumerable<List<Notification>> CurrentUser(User userId)
```

Comment:
IEnumerable\<List\<Notification>> !? IEnumerable of Lists of Notifications? .... no.

And "CurrentUser"? The suggests that this method returns the current user.